clang/test/Sema/builtin-get-counted-by.c

```diff
@@ -0,0 +1,28 @@
+// RUN: %clang_cc1 -fsyntax-only -verify %s
+
```

Comment:
I also only thought of this just now, but some tests using `_Generic` to verify that the return type of `__builtin_get_counted_by` is indeed what it’s supposed to be would be nice.

Reply:
I've never heard of `_Generic` before. What is it?

Reply:
A C11 feature to select between expressions based on the type of some controlling expression:
https://en.cppreference.com/w/c/language/generic

Reply:
never mind. I discovered what it's for. Added. :)